app/drivers/kscan/Kconfig

```diff
@@ -14,6 +14,27 @@ config ZMK_KSCAN_MATRIX_POLLING
 config ZMK_KSCAN_DIRECT_POLLING
 	bool "Poll for key event triggers instead of using interrupts on direct wired boards."
 
+config ZMK_KSCAN_GLOBAL_DEBOUNCE_PRESS
+	bool
```

Comment:
This should get a string label as well, e.g.


```suggestion
	bool "Global kscan debounce support"
```

Reply:
I think I forgot to remove this. It doesn't appear to be used anywhere.